src/Vegetation/BigLeaf/bigleaf.jl

```diff
@@ -0,0 +1,120 @@
+module BigLeaf
+using ClimaLSM
+using UnPack # do we need?
```

Comment:
we can remove! we can unpack structs now like (;a, b) = struct, or similar to that.